packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/creditCard.js

```diff
@@ -0,0 +1,273 @@
+import React, { useMemo } from 'react';
+import { bool, func, shape, string } from 'prop-types';
+import { useCreditCard } from '@magento/peregrine/lib/talons/CheckoutPage/PaymentInformation/useCreditCard';
+
+import combine from '../../../util/combineValidators';
+import {
+    hasLengthExactly,
+    isRequired,
+    validateRegionCode
+} from '../../../util/formValidators';
+import Country from '../../Country';
+import Region from '../../Region';
+import Checkbox from '../../Checkbox';
+import Field from '../../Field';
+import TextInput from '../../TextInput';
+import BrainTreeDropin from './brainTreeDropIn';
+import LoadingIndicator from '../../LoadingIndicator';
+import { mergeClasses } from '../../../classify';
+
+import creditCardPaymentOperations from './creditCard.gql';
+
+import defaultClasses from './creditCard.css';
+
+const STEP_DESCRIPTIONS = [
+    /**
+     * This is because step numbers start from 1
+     * but array indexes start from 0. Hence setting
+     * index 0 to `null`.
+     */
+    null,
+    'Checking Credit Card Information',
+    'Checking Credit Card Information',
+    'Checking Credit Card Information',
+    'Checking Credit Card Information'
```

Comment:
Is this array still necessary anymore? Now all the steps result in the same text being rendered.

Reply:
Done 👍 

Changed to something more meaningful.